prodigy.el

```diff
@@ -1274,8 +1274,10 @@ SIGNINT signal."
 ;;;; View mode functions
 
 (defun prodigy-strip-ctrl-m (output)
-  "Strip 
 line endings from OUTPUT."
-  (s-replace "
" "" output))
+  "Strip 
+ line endings from OUTPUT."
+  (s-replace "
+" "" output))
```

Comment:
The diff is really weird as well because the `s-replace` part is already there in the actual code. I don't get it...? 😄